ygot/render_test.go

```diff
@@ -1658,6 +1658,51 @@ type exampleTransportAddressBinary struct {
 
 func (*exampleTransportAddressBinary) IsExampleTransportAddress() {}
 
+// ucExampleDevice and the following structs are a set of structs used for more
+// complex testing in TestConstructIETFJSON, these structs are used to test for
+// YANG presence containers.
+
+type ucExampleDevice struct {
+	Bgp *ucExampleBgp `path:"bgp" yangPresence:"true"`
```

Comment:
I'm also thinking of the cases of a presence container within a non-presence container or a list entry. I think they'll work but a test would guarantee it.

Reply:
done!